libraries/standard/mqtt/include/iot_mqtt.h

```diff
@@ -148,6 +148,12 @@ void IotMqtt_ReceiveCallback( IotNetworkConnection_t pNetworkConnection,
  * Any restored subscriptions <b>MUST</b> have been present in the persistent session;
  * <b>this function does not send an MQTT SUBSCRIBE packet!</b>
  *
+ * [pConnectInfo->pPreviousSubscriptions](@ref IotMqttConnectInfo_t.pPreviousSubscriptions)
+ * and [pConnectInfo->previousSubscriptionCount](@ref IotMqttConnectInfo_t.previousSubscriptionCount) can
+ * also be used to pass a list of subscriptions to be stored locally without a SUBSCRIBE packet being
+ * sent to broker. These subscriptions are useful to invoke application level callbacks for messages received
+ * on unsolicited topics from broker.
```

Comment:
The following comment applies throughout: 

Is receiving unsolicited messages supported in the MQTT spec?  I couldn't find any reference to this in the MQTT specification.  If this is a feature specific to a particular MQTT broker, it should be documented as such.

Reply:
This is covered in [Section 4.5](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc398718104).